scripts/validate-android-test-env.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+# This script validates that Android is set up correctly for the
+# testing environment.
+#
+# In particular, the config in ReactAndroid/build.gradle must match
+# the android sdk that is actually installed. Also, we must have the
+# right version of Java.
+
+# Check that Buck is working.
+if [ -z "$(which buck)" ]; then
+  echo "You need to install Buck."
+  echo "See https://buckbuild.com/setup/install.htm for instructions."
+  exit 1
+fi
+
+if [ -z "$(buck --version)" ]; then
+  echo "Your Buck install is broken."
+  if [ -d "/opt/facebook" ]; then
+    echo "FB laptops ship with a Buck config that is not compatible with open "
+    echo "source. FB Buck requires the environment to set a buck version, but "
+    echo "the open source version of Buck forbids that. Try setting:"
+    echo "export BUCKVERSION=ff27d5270ecaa92727cd5a19954e62298fa78f09"
```

Comment:
I'd really like to just include a `.buckversion` in the repo but since FB Buck isn't compatible with open source Buck the "right way" doesn't work here. Is there a better way? I would like to provide a specific Buck version that works because I did run into incomprehensible failures when using some Buck versions.